lua/lib/populate.lua

```diff
@@ -15,6 +15,8 @@ local path_to_matching_str = require'lib.utils'.path_to_matching_str
 local function dir_new(cwd, name)
   local absolute_path = cwd..'/'..name
   local stat = luv.fs_stat(absolute_path)
+  local handle = luv.fs_scandir(absolute_path)
```

Comment:
I'm getting an error here when opening nvim in my home dir. `absolute_path` is nil `fs_scandir` doesn't like that.

Reply:
how could it be nil ? It should also fail the `fs_stat` call in this case, this might be related to something else.

Reply:
https://github.com/kyazdani42/nvim-tree.lua/pull/207 fixes my problem

Reply:
Somehow missed these comments when fixing the problem, but thank you for pointing it out @stsewd !